### DISCUSSION TITLE
我使用onnxruntime对齐官方的推理，推理速度比使用paddlepaddle框架推理快很多（速度快5，6倍）

### DISCUSSION THREAD

**Discussion by jingsongliujing** [PaddlePaddle/PaddleOCR#13195] [Q&A] [answered]
#### 问题描述 / Problem Description

我使用onnxruntime对齐官方的推理，推理速度比使用paddlepaddle框架推理快很多（速度快5，6倍），是框架有问题吗
#### 运行环境 / Runtime Environment
- windows/linux:
-onnxruntime

#### 复现代码 / Reproduction Code
代码：https://github.com/jingsongliujing/OnnxOCR.git

**Comment by GreatV** [comment]
是比 paddle inference 快吗

**Comment by jingsongliujing** [comment]
> 是比 paddle inference 快吗

是的，要快很多

**Comment by jingsongliujing** [comment]
特别是v4版本

**Comment by jingsongliujing** [comment]
其他几个模型不明显，就v4 mobile 比较明显

**Comment by GreatV** [comment]
用 `onnx=True` 来跑呢

**Comment by jingsongliujing** [comment]
没试过，但是官方框架它本身会加载很多配置，估计推理速度也会比自己拆出来慢一些

**Comment by jingsongliujing** [comment]
我目前只做了检测，分类，识别三个模型的串联，表格，抽取啥的没考虑

**Reply by MingsYang** [reply]
请问效果和资源占用上有差异吗

**Comment by SWHL** [comment]
这和我们做的[RapidOCR](https://github.com/RapidAI/RapidOCR)似乎是一样的

**Reply by jingsongliujing** [reply]
> 这和我们做的[RapidOCR](https://github.com/RapidAI/RapidOCR)似乎是一样的

嗯，看了一下，确实差不多，但是我看仓库只更新到了v3的onnx版本，v4是否有更新呢

**Reply by SWHL** [reply, marked as answer]
有的，目前v1.3.22里面集成的就是v4模型。你看到的应该是目录名是v3的。

**Comment by Fire-Star** [comment]
onnx c++，gpu  服务端模型v4，推理速度 7ms 识别单行文本10个字，3060
